x/chainlink/types/msg.go

```diff
@@ -74,32 +73,36 @@ func NewModuleOwner(assigner, address sdk.Address, pubKey []byte) *ModuleOwner {
 	return mo
 }
 
-func (m *ModuleOwner) Route() string {
+func (m *MsgModuleOwner) Route() string {
 	return RouterKey
 }
 
-func (m *ModuleOwner) Type() string {
+func (m *MsgModuleOwner) Type() string {
 	return AddModuleOwner
 }
 
-func (m *ModuleOwner) ValidateBasic() error {
+func (m *MsgModuleOwner) ValidateBasic() error {
 	// TODO: add proper cosmos address and pubkey validation
 	return nil
 }
 
-func (m *ModuleOwner) GetSignBytes() []byte {
+func (m *MsgModuleOwner) GetSignBytes() []byte {
 	bz := ModuleCdc.MustMarshalJSON(m)
 	return sdk.MustSortJSON(bz)
 }
 
-func (m *ModuleOwner) GetSigners() []sdk.AccAddress {
+func (m *MsgModuleOwner) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{sdk.AccAddress(m.AssignerAddress)}
 }
 
-type ModuleOwners []*ModuleOwner
+func (m *MsgModuleOwner) GetMsgs() []githubcosmossdktypes.Msg {
```

Comment:
not quite sure what this is used for,,

Reply:
this is used in anteHandler to parse txs from Msg and do type casting